unified-runtime/test/conformance/lit.cfg.py

```diff
@@ -7,5 +7,50 @@
 
 """
 
-# Disable lit testing (for now)
-config.suffixes = []
+import lit.formats
+import subprocess
+from os import path
+
+config.name = "Unified Runtime Conformance"
+config.test_format = lit.formats.GoogleTest("", "-test")
+config.test_source_root = config.binary_dir
+config.test_exec_root = config.binary_dir
+
+config.environment["UR_LOADER_USE_LEVEL_ZERO_V2"] = "1" if config.using_l0_v2 else "0"
```

Comment:
What if both adapters (legacy and v2) are built?

Reply:
Only L0 V2 will be ran. I think this is an unavoidable change with how we are changing the structure of tests; we are switching to having the test binaries run tests for all devices rather than overriding the loaded shared object. This change shouldn't affect CI (since the "L0 v1" job doesn't build v2).

I should have mentioned it in the commit message/PR description though, thanks for pointing it out.